clustered/server/src/main/java/org/ehcache/clustered/server/EhcacheActiveEntity.java

```diff
@@ -309,101 +188,21 @@ private void clearClientTrackedAtReconnectComplete() {
-      inflightInvalidations.compute(cacheId, (s, tuples) -> {
-        if (tuples == null) {
-          tuples = new ArrayList<>();
-        }
-        tuples.add(new InvalidationTuple(clientDescriptor, invalidationsInProgress, reconnectMessage.isClearInProgress(cacheId)));
-        return tuples;
-      });
-    }
-  }
-
   @Override
   public void synchronizeKeyToPassive(PassiveSynchronizationChannel<EhcacheEntityMessage> syncChannel, int concurrencyKey) {
-    LOGGER.info("Sync started for concurrency key {}.", concurrencyKey);
-    if (concurrencyKey == DEFAULT_KEY) {
-      ServerSideConfiguration configuration;
-      if (ehcacheStateService.getDefaultServerResource() == null) {
-        configuration = new ServerSideConfiguration(ehcacheStateService.getSharedResourcePools());
-      } else {
-        configuration = new ServerSideConfiguration(ehcacheStateService.getDefaultServerResource(), ehcacheStateService.getSharedResourcePools());
-      }
-
-      Map<String, ServerStoreConfiguration> storeConfigs = new HashMap<>();
-      for (String storeName : ehcacheStateService.getStores()) {
-        ServerSideServerStore store = ehcacheStateService.getStore(storeName);
-        storeConfigs.put(storeName, store.getStoreConfiguration());
-      }
-
-      syncChannel.synchronizeToPassive(new EhcacheStateSyncMessage(configuration, storeConfigs));
-    } else {
-      Long dataSizeThreshold = Long.getLong(SYNC_DATA_SIZE_PROP, DEFAULT_SYNC_DATA_SIZE_THRESHOLD);
-      AtomicLong size = new AtomicLong(0);
-      ehcacheStateService.getStores()
-        .forEach(name -> {
-          ServerSideServerStore store = ehcacheStateService.getStore(name);
-          final AtomicReference<Map<Long, Chain>> mappingsToSend = new AtomicReference<>(new HashMap<>());
-          store.getSegmentKeySets().get(concurrencyKey - DATA_CONCURRENCY_KEY_OFFSET)
-            .forEach(key -> {
-              final Chain chain;
-              try {
-                chain = store.get(key);
-              } catch (TimeoutException e) {
-                throw new AssertionError("Server side store is not expected to throw timeout exception");
-              }
-              for (Element element : chain) {
-                size.addAndGet(element.getPayload().remaining());
-              }
-              mappingsToSend.get().put(key, chain);
-              if (size.get() > dataSizeThreshold) {
-                syncChannel.synchronizeToPassive(new EhcacheDataSyncMessage(name, mappingsToSend.get()));
-                mappingsToSend.set(new HashMap<>());
-                size.set(0);
-              }
-            });
-          if (!mappingsToSend.get().isEmpty()) {
-            syncChannel.synchronizeToPassive(new EhcacheDataSyncMessage(name, mappingsToSend.get()));
-            mappingsToSend.set(new HashMap<>());
-            size.set(0);
-          }
-        });
-    }
-    LOGGER.info("Sync complete for concurrency key {}.", concurrencyKey);
+    // Nothing to sync
```

Comment:
Nothing to sync? What about the default server resource configuration and the pool definitions?